### ISSUE TITLE
[FavoriteMedia]: plugin couldn't be compiled error

### ISSUE THREAD

**Issue by thestexe** [Dastan21/BDAddons#282]
### Discord Build

Stable

### Discord Version

stable 352804 (965be3c)

### Plugin Version

1.11.1

### Steps To Reproduce

1. install FavoriteMedia from plugin repo
2. this error from screenshot pops up
3. plugin can't be turned on

### Describe the bug

error on install

### Screenshots

![Screenshot 2024-12-12 141518](https://github.com/user-attachments/assets/7f7e652d-69d0-4ba5-a2ea-d47017481d42)

**Comment by bob-carter** [comment]
Yep, new API update, and it's recommend plugins move off of Zera to the new API from BD directly. 
On the plus side the API should be more stable in the future.

**Comment by foursplaysroblox** [comment]
also happened in ptb

**Comment by Weigurde** [comment]
is this getting fixed soon?

**Comment by Dastan21** [comment]
Working on it, please be patient.
